fds/services/dvc_service.py

```diff
@@ -333,7 +333,15 @@ def pull(self, git_url: str, remote_name: Optional[str]) -> Any:
         :param remote_name: Optional Remote dvc name to pull the dvc repository
         :return:
         """
-        self.printer.warn("Staring DVC Clone...")
+        # Check if dvc clone can happen
+        if not does_file_exist(".dvc"):
```

Comment:
You need to do this relative to the dvc root I think

Reply:
Yes I have handled this in other PR, now that its merged I'll fix this